src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -38,12 +42,14 @@
    */
   private static final Item.Properties SMELTERY_PROPS = new Item.Properties().group(TinkerRegistry.tabSmeltery);
   private static final Function<Block,? extends BlockItem> TOOLTIP_BLOCK_ITEM = (b) -> new BlockTooltipItem(b, SMELTERY_PROPS);
+  private static final Function<Block,? extends BlockItem> TANK_BLOCK_ITEM = (b) -> new TankItem(b, SMELTERY_PROPS);
```

Comment:
No need to make this a varaible, I only did that because it was reused, this constructor is only used once.

Reply:
Ah yeah. I made the variable before I turned three tanks into one enum
object.